tls_icon.php

```diff
@@ -5,6 +5,7 @@ class tls_icon extends rcube_plugin
 	const POSTFIX_TLS_REGEX = "/\(using (TLS.*)\) \(/im";
 	const POSTFIX_LOCAL_REGEX = "/\([a-zA-Z]*, from userid [0-9]*\)/im";
 	const SENDMAIL_TLS_REGEX = "/\(version=(TLS.*)\)(\s+for|;)/im";
+    const STALWART_TLS_REGEX = "/\(using (TLSv1\.[23] with cipher .+)\) by .+ \(Stalwart SMTP\)/im";
```

Comment:
For reference the name seems to be hard coded: https://github.com/stalwartlabs/mail-server/blob/24df9a0352a948f67ad58ab0d7d7423002863d43/crates/smtp/src/inbound/data.rs#L947

But I do not see the point of keeping it in the regex
```suggestion
    const STALWART_TLS_REGEX = "/\(using (TLSv1\.[23] with cipher .+)\)/im";
```